File: 프로그래머스/0/181855. 문자열 묶기/문자열 묶기.cs
using System;
using System.Linq;
public class Solution {
    public int solution(string[] strArr) {
        int[] arr = new int[31];
        int max = 0;
        for(int i = 0 ; i < strArr.Length; i++)
        {
            arr[strArr[i].Length] += 1;
        }
        max = arr.Max();
        return max;
    }
}

File: 프로그래머스/0/181855. 문자열 묶기/문자열 묶기.cpp
#include <string>
#include <vector>
#include <unordered_map>
using namespace std;

int solution(vector<string> strArr) {
    int length = 0;
    int maxvalue = 0;
    unordered_map<int,int> map;
    for(auto elem : strArr)
    {
        length = elem.length();
        map[length] = map[length] + 1;
        
        if(map[length] > maxvalue)
            maxvalue = map[length];
    }
    return maxvalue;
}